addons/bus/models/bus_presence.py

```diff
@@ -31,6 +33,21 @@ class BusPresence(models.Model):
 
     @api.model
     def update(self, inactivity_period):
+        try:
+            self.with_context(MAX_TRIES_ON_CONCURRENCY_FAILURE=0)._update(inactivity_period)
```

Comment:
I don't understand how this could work, as it does not modify `request.env` in the caller.
Can you explain?

Reply:
You are right, it doesn't modify request.env